src/query/src/sql.rs

```diff
@@ -590,6 +599,31 @@ pub fn show_create_table(
     Ok(Output::new_with_record_batches(records))
 }
 
+pub fn show_create_flow(obj_name: &ObjectName, flow_val: FlowInfoValue) -> Result<Output> {
```

Comment:
We have a `CreateFlow` statement that implements Display.
https://github.com/GreptimeTeam/greptimedb/blob/c78043d526134b6cad19151e5ffd25aaefc778fe/src/sql/src/statements/create.rs#L267-L287

You could construct that statement and then convert it to a string. This is how `SHOW CREATE TABLE` works.
https://github.com/GreptimeTeam/greptimedb/blob/main/src/query/src/sql/show_create_table.rs